tests/e2e/kafkatopic_webhook.go

```diff
@@ -0,0 +1,333 @@
+// Copyright © 2023 Cisco Systems, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+func testWebhookKafkaTopic(kafkaCluster types.NamespacedName) {
```

Comment:
**Suggestion**
We dont need to use any arguments here for testWebhookKafkaTopic because the namespace and kafkacluster name is come always from const variable. If later we want to change that then we need to change almost for all of the other tests.
Because of the consistency we should remove this.

Reply:
I see what you mean. Consistency is a stronger argument.

Does this commit address it correctly:  ee82a868b1eff93600c31bfe66fd952663579356 ?